### ISSUE TITLE
[Accessibility][Sourcing] Accessibility issue: fd-list inside fd-popover-body doesnt work on hitting enter key

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#6156]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
I am using  "@fundamental-ngx/core": "^v0.31.0-rc.125",
for the component fd-popover, referring to the link https://fundamental-ngx.netlify.app/#/core/popover
` <fd-popover-body *ngIf="list1 && list1.length">
            <ul fd-list>
                <li fd-list-item *ngFor="let option of list1">
                    <a [attr.href]="option.url" fd-list-title>{{ option.text }}</a>
                </li>
            </ul>
        </fd-popover-body>`
On hitting enter key when focus is on any option , it doesnt navigate to the option.url but works only on mouse click.
Its more on accessibility issue. Please fix it.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
"@fundamental-ngx/core": "^v0.31.0-rc.125", angular 10
#### If this is a bug, please provide steps for reproducing it.
1) Get the focus on the fd-popover trigger button , hit enter 
2) opens popover body
3) default focus is on the 1st option li 
4) hit enter on it 
5) nothing happens , expected behaviour: should have navigated to some link 
#### Please provide relevant source code if applicable.
`  <fd-popover [focusTrapped]="true" [focusAutoCapture]="true" title="photo 1 in circle">
        <fd-popover-control>
            <fd-avatar size="s" [circle]="true" ariaLabel="photo 1 in circle" image="http://picsum.photos/id/1018/400"></fd-avatar>
        </fd-popover-control>
        <fd-popover-body *ngIf="list1 && list1.length">
            <ul fd-list>
                <li fd-list-item *ngFor="let option of list1">
                    <a [attr.href]="option.url" fd-list-title>{{ option.text }}</a>
                </li>
            </ul>
        </fd-popover-body>
    </fd-popover>`
#### Is there anything else we should know?
As an alternate i am currently adding tabindex=0 and (keydown.enter)="option.url" directly on li tag

**Comment by skaquib123** [comment]
@I543348 , If you are using **fd-list**  in <ul> then you have to use **fd-list-item** in <li> and **fd-checkbox** not fdp-checkbox. There is a compatibility issue between fd and fdp components.
